store/sqlstore/command_webhook_store.go

```diff
@@ -47,33 +48,31 @@ func (s SqlCommandWebhookStore) Save(webhook *model.CommandWebhook) (*model.Comm
 	}
 
 	if err := s.GetMaster().Insert(webhook); err != nil {
-		return nil, model.NewAppError("SqlCommandWebhookStore.Save", "store.sql_command_webhooks.save.app_error", nil, "id="+webhook.Id+", "+err.Error(), http.StatusInternalServerError)
+		return nil, errors.Wrapf(err, "Save id=%s", webhook.Id)
```

Comment:
I think what I had been doing is to just mention the method name with a colon at end: `insert: id=%s`. But we have also been explaining the action in some PRs: `failed to insert webhook with id=%s`. We need a PR to just use a single convention. I am fine with whatever you want to use in this PR. But let's choose any one.